Tangem/Modules/ManageTokens/ManageTokensViewModel.swift

```diff
@@ -10,4 +10,160 @@ import Foundation
 import SwiftUI
 import Combine
 
-final class ManageTokensViewModel: ObservableObject {}
+final class ManageTokensViewModel: ObservableObject {
+    // MARK: - Injected & Published Properties
+
+    @Injected(\.tokenQuotesRepository) private var tokenQuotesRepository: TokenQuotesRepository
+    @Injected(\.userWalletRepository) private var userWalletRepository: UserWalletRepository
+
+    // I can't use @Published here, because of swiftui redraw perfomance drop
+    var enteredSearchText = CurrentValueSubject<String, Never>("")
+
+    @Published var tokenViewModels: [ManageTokensItemViewModel] = []
+    @Published var isLoading: Bool = true
+    @Published var alert: AlertBinder?
+    @Published var showToast: Bool = false
+
+    // MARK: - Properties
+
+    var hasNextPage: Bool {
+        loader.canFetchMore
+    }
+
+    private lazy var loader = setupListDataLoader()
+
+    private var bag = Set<AnyCancellable>()
+    private var loadQuotesSubscribtion: AnyCancellable?
+
+    private unowned let coordinator: ManageTokensRoutable
+
+    private var percentFormatter = PercentFormatter()
+    private var balanceFormatter = BalanceFormatter()
```

Comment:
Я бы чуть переставил. Ну или хотя бы надо `var` у форматов убрать 
```suggestion
    private unowned let coordinator: ManageTokensRoutable

    private lazy var loader = setupListDataLoader()
    private let percentFormatter = PercentFormatter()
    private let balanceFormatter = BalanceFormatter()
    private var bag = Set<AnyCancellable>()
    private var loadQuotesSubscribtion: AnyCancellable?
```